### ISSUE TITLE
batch size for pretraining

### ISSUE THREAD

**Issue by xiaovhua** [salesforce/BLIP#42]
Hi! Thank you for your amazing work on VLP. We are now implementing the pretraining of BLIP, but we notice that the batch size of your code(75*8=600 for VIT-B) is unmatched with the paper(2880 for VIT-B). Since no ablation experiment on batch size is offered, we want to know whether the performance will be better with larger batch size if training data is sufficient?

**Comment by LiJunnan1992** [comment]
Hi, I used 90*32=2880 for training ViT-B. The batch size will not have a big impact on the performance and a batch size of 600 should work well. Note that the learning rate may need to be smaller if the batch size is smaller.

**Comment by xiaovhua** [comment]
> Hi, I used 90*32=2880 for training ViT-B. The batch size will not have a big impact on the performance and a batch size of 600 should work well. Note that the learning rate may need to be smaller if the batch size is smaller.

I see! Thanks for your kind reply!
